src/pages/learn.js

```diff
@@ -70,6 +71,14 @@ const HeaderContent = styled('div')`
     max-width: ${FEATURED_ARTICLE_MAX_WIDTH};
     margin-left: auto;
     margin-right: auto;
+    ${H1} {
```

Comment:
We can take a different approach that won't require us to re-do the H2 styling here.

Let's keep importing H2 from the text module, and we can use this line in the component to render it with an `h1` tag as such:

```
const H2asH1 = H2.withComponent('h1');
```

then when we render, we use this new tag so we can do:

```
<H2asH1>Make better, faster applications</H2asH1>
```

and similar for below, does this make sense?

Reply:
Yeah, looks better and clear, thanks for a tip.